packages/task/src/browser/task-frontend-contribution.ts

```diff
@@ -132,6 +138,14 @@ export class TaskFrontendContribution implements CommandContribution, MenuContri
                 execute: () => this.taskService.runSelectedText()
             }
         );
+
+        registry.registerCommand(
+            TaskCommands.TASK_CONFIGURE,
+            {
+                isEnabled: () => true,
```

Comment:
bwt `isEnabled` is `true` by default as well as `isVisible`

Reply:
Removed